src/main/resources/terraform/hello-world-config.tf

```diff
@@ -0,0 +1,9 @@
+provider "docker" {
+}
+resource "docker_image" "ubuntu" {
+  name = "ubuntu:precise"
```

Comment:
`ubuntu` image does not start a service, and by default exits immediately.
Better example would be `nginx` - container will continue working, we can check its status by opening a web page